WordPress/Classes/Models/ReaderTagTopic.swift

```diff
@@ -4,6 +4,7 @@ import Foundation
     @NSManaged open var isRecommended: Bool
     @NSManaged open var slug: String
     @NSManaged open var tagID: NSNumber
+    @NSManaged open var card: NSOrderedSet?
```

Comment:
Should't this be cards if its a set?

Reply:
Yes!

Done in 3a6cfe67a1ae21593e87fda24956511dd94309e1